lib/api.js

```diff
@@ -14,9 +14,8 @@ import _ from 'lodash';
 var api = new base.API({
   title:        "TaskCluster Secrets API Documentation",
   description: [
-    "The secrets service, typically available at",
-    "`tools.taskcluster.net`, is responsible for managing",
-    "secure data in TaskCluster."
```

Comment:
> I think there is a lot of value to having self-describing end-points for discoverability.

I agree... It would be nice... However, we should probably just do the documentation service we talked about and then do a 404 page containing a link to the docs... That would be fairly easy... Self-describing APIs would be the right way to do REST, but it's also a significant investment. I prefer consistent docs using reference formats as we have now.

@petemoore, honestly, if you want to polish all of this... I think you'll have to undertake it. But I think we have more important things right now. That said it's possible we should do a refactor of all the glue code some day.